src/dashboard/components/StatsGrid/StatsGrid.tsx

```diff
@@ -0,0 +1,70 @@
+import { Group, Paper, SimpleGrid, Text } from '@mantine/core';
+import {
+  IconUserPlus,
+  IconDiscount2,
+  IconReceipt2,
+  IconCoin,
+  IconArrowUpRight,
+  IconArrowDownRight,
+  IconEqual,
+} from '@tabler/icons-react';
+import classes from './StatsGrid.module.css';
+
+const icons = {
+  user: IconUserPlus,
+  discount: IconDiscount2,
+  receipt: IconReceipt2,
+  coin: IconCoin,
+};
+
+export interface TransactionStatsProps {
+  statistics: {
+    title: string;
+    diff: number;
+    icon: string;
+    value: number;
+  }[];
+}
+
+const StatsGrid: React.FC<TransactionStatsProps> = ({ statistics }) => {
+  const stats = statistics.map((stat) => {
```

Comment:
Might want to wrap this in a useMemo if `statistics` is going to be a big obj (or memoize the component itself)

Reply:
No stress on this I think, don't plan to have statistics as some some big object (at most like 20 items within the list)